src/main/java/org/terasology/signalling/blockFamily/SignalCableBlockFamilyFactory.java

```diff
@@ -180,7 +194,13 @@ private Rotation getRotationToAchieve(byte source, byte target) {
         }
         return null;
     }
-    
+
+    /**
+     * Processes updates for a given {@link BlockComponent}
+     *
+     * @param event A {@link OnBlockItemPlacedEvent} to fetch the {@link BlockComponent}
+     * @param entity An ignored entity
+     */
     @ReceiveEvent(components = {BlockItemComponent.class})
     public void onPlaceBlock(OnBlockItemPlaced event, EntityRef entity) {
```

Comment:
Anything with an `@ReceiveEvent` is a listener that is triggered when the event is sent. It's probably better to rephrase this to "Processes updates for the block's neighbours when the block is placed"